Classes/Command/ProjectionCommandController.php

```diff
@@ -116,11 +116,16 @@ public function replayCommand($projection, $quiet = false)
             $this->output->progressStart();
         }
         $eventsCount = 0;
-        $this->projectionManager->replay($projectionDto->getIdentifier(), function () use (&$eventsCount, $quiet) {
+        $startOfLastThousandEvents = microtime(true);
+        $this->projectionManager->replay($projectionDto->getIdentifier(), function () use (&$eventsCount, $quiet, &$startOfLastThousandEvents) {
             $eventsCount ++;
             if (!$quiet) {
                 $this->output->progressAdvance();
             }
+            if ($eventsCount % 1000 === 0) {
+                $this->outputLine('Time taken for 1000 events: ' . (microtime(true) - $startOfLastThousandEvents));
+                $startOfLastThousandEvents = microtime(true);
```

Comment:
Actually I think that the output "time taken for …" is only interesting now that you were tackling speed, but in the long run we shouldn't care - it should just be fast. I'd remove this output or introduce a super-nerdy "--display-profiling-data" option.